templates/zerver/help/custom-certificates.md

```diff
@@ -35,6 +41,62 @@ administrator.
 2. Select the **Organizations** tab.
 
 3. Under **Add Custom Certificates**, enter your organization URL and add
-   the custom certificate file (should end in `.crt` or `.pem`).
+   the custom certificate file (it should end in `.crt` or `.pem`).
 
 {end_tabs}
+
+
+# Adding a certificate
+
+A file containing the certificate will need to be provided by your
+administrator; it should be a file ending in `.pem` or `.crt`.
+
+
+{start_tabs}
+{tab|mac}
+1. Hit `Cmd+Space` to bring up Spotlight Search, type `Keychain
```

Comment:
Consider using `**bold**` rather than `` `code` `` for most of these non-code labels.

Reply:
Looks like our current convention is no formatting on keypresses (though we could potentially decide to use [an extension for that](https://facelessuser.github.io/pymdown-extensions/extensions/keys/#options) if we thought it reasonable enough to steal `++`), and bold for other things; will swap that up.